projects/01/DMux.hdl

```diff
@@ -14,5 +14,7 @@ CHIP DMux {
     OUT a, b;
 
     PARTS:
-    // Put your code here:
+    Not(in=sel, out=notsel);
+    And(a=in, b=notsel, out=a);
+    And(a=in, b=sel, out=b);

```

Comment:
Similarly:

in | sel | in && !sel | in && sel
--|-----|-----------|----------
0 | 0 | 0 (= in) | 0
1 | 0 | 1 (= in) | 0
0 | 1 | 0 | 0 (= in)
1 | 1 | 0 | 1 (= in)